modules/serverless-open-telemetry.adoc

```diff
@@ -0,0 +1,149 @@
+* You have not yet installed the {ServerlessOperatorName} and Knative Serving. These must be installed after the {DTProductName} installation.
+* You have installed {DTProductName} by following the {product-title} "Installing distributed tracing" documentation.
+* Install the OpenShift (`oc`) CLI.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+
+.Procedure
+
+. Create an `OpenTelemetryCollector` custom resource (CR):
++
+.Example OpenTelemetryCollector CR
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: tracing-system
+spec:
+  mode: deployment
+  config: |
+    receivers:
+      zipkin:
+    processors:
+    exporters:
+      jaeger:
+        endpoint: jaeger-all-in-one-inmemory-collector-headless.tracing-system.svc:14250
+        tls:
+          ca_file: "/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+      logging:
+    service:
+      pipelines:
+        traces:
+          receivers: [zipkin]
+          processors: []
+          exporters: [jaeger, logging]
+----
+
+. Verify that you have two pods running in the namespace where {DTProductName} is installed:
++
+[source,terminal]
+----
+$ oc get pods -n <namespace>
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                          READY   STATUS    RESTARTS   AGE
+cluster-collector-collector-85c766b5c-b5g99   1/1     Running   0          5m56s
+jaeger-all-in-one-inmemory-ccbc9df4b-ndkl5    2/2     Running   0          15m
+----
+
+. Verify that the following headless services have been created:
++
+[source,terminal]
+----
+$ oc get svc -n tracing-system | grep headless
+----
++
+.Example output
+[source,terminal]
+----
+cluster-collector-collector-headless            ClusterIP   None             <none>        9411/TCP                                 7m28s
+jaeger-all-in-one-inmemory-collector-headless   ClusterIP   None             <none>        9411/TCP,14250/TCP,14267/TCP,14268/TCP   16m
+----
++
+These services are used in to configure Jaeger and Knative Serving. The name of the Jaeger service may vary.
+
+. Install the {ServerlessOperatorName} by following the "Installing the {ServerlessOperatorName}" documentation.
+
+. Install Knative Serving by creating the following `KnativeServing` CR:
++
+.Example KnativeServing CR
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+    name: knative-serving
+    namespace: knative-serving
+spec:
+  config:
+    tracing:
+      "backend": "zipkin"
+      "zipkin-endpoint": "http://cluster-collector-collector-headless.tracing-system.svc:9411/api/v2/spans"
+      "debug": "true"
+      sample-rate: "1" <1>
+----
+<1> The sample rate determines what percentage of traces are stored for visualization.
```

Comment:
Actually we should use the same description as in [here](https://docs.openshift.com/container-platform/4.10/serverless/monitor/serverless-tracing.html#serverless-jaeger-config_serverless-tracing) to be consistent, although at the end of the day it describes the same thing. It is probabilistic sampling. Value 1 here is for demonstration reasons to make sure all traces are stored. In a real setup you want to store less to get to avoid storage overhead. In [istio integration](https://istio.io/latest/docs/tasks/observability/distributed-tracing/jaeger/) for example default is 1%.

"The sample-rate defines sampling probability. Using sample-rate: "0.1" means that 1 in 10 traces will be sampled."